client/scraper/metadata.py

```diff
@@ -58,6 +59,19 @@ def parse_content(self, response):
 
     def to_json(self):
         json_data = json.dumps(self.prop_map)
+        self.cache_metadata(json_data)
+        response = FlaskResponse(response=json_data, status=StatusCode.OK, mimetype="application/json")
+        return response
+
+    def cache_metadata(self, json_data):
+        try:
+            db_utils.DbInstance.db.add_value(self.prop_map[FieldKeyword.URL], json_data)
+        except db_utils.MagpieDbError:
+            # TODO: Log database error
+            return
+
+    @staticmethod
+    def create_response_from_db(json_data):
```

Comment:
this should not be in the Metadata class.
It should be in db_utils. I'm also not sure why this needs to be a static method